pycqed/instrument_drivers/meta_instrument/LutMans/mw_lutman.py

```diff
@@ -745,7 +735,10 @@ def apply_mixer_predistortion_corrections(self, wave_dict):
         M_D = wf.mixer_predistortion_matrix(self.D_mixer_alpha(),
                                             self.D_mixer_phi())
 
+        print(self.LutMap())
```

Comment:
These print statements should be removed.

Reply:
Will do.